src/common/copilot/PowerPagesCopilot.ts

```diff
@@ -202,6 +202,10 @@ export class PowerPagesCopilot implements vscode.WebviewViewProvider {
           this.openWalkthrough();
           break;
         }
+        case "codeLineCount": {
+          sendTelemetryEvent(this.telemetry, { eventName: CopilotCodeLineCountEvent, copilotSessionId: sessionID, codeLineCount: data.value });
```

Comment:
I am concerned with impact on telemetry of the new codeLineCount event

Reply:
A single event will be logged for each copilot response.